linux/keyman-config/keyman_config/view_installed.py

```diff
@@ -175,19 +175,19 @@ def addlistitems(self, installed_kmp, store, install_area):
 
             if install_area == InstallArea.IA_USER:
                 welcome_file = os.path.join(user_keyboard_dir(kmpdata['packageID']), "welcome.htm")
-                icofile = os.path.join(user_keyboard_dir(kmpdata['packageID']), kmpdata['packageID'] + ".ico.png")
+                icofile = os.path.join(user_keyboard_dir(kmpdata['packageID']), kmpdata['packageID'] + ".png")
```

Comment:
I'm not sure it's safe to use the package name + .png because there may be packages out there that already contain a file with that name...

Reply:
I searched the keyboards repo and found these source/png's that aren't icon files:

./release/nlci/nlci_malayalam_winscript/source/nlci_malayalam_winscript.png
./release/s/swanalekha_malayalam/source/swanalekha_malayalam.png

Since the image conversions go `.ico` --> `.bmp` --> `.png`, maybe I'll use `.bmp.png` for file extension.

Reply:
Here's how the installs look using `.bmp.png`
![Screenshot from 2019-09-12 10-10-29](https://user-images.githubusercontent.com/7358010/64751199-a3dadc80-d545-11e9-84f2-80b4f3ae1447.png)

Reply:
Sounds good to me. Good research work there too :)